<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>加密端服务</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>436</width>
     <height>722</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Encryption</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>File</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="4">
     <widget class="QTextEdit" name="EnPlaintext">
      <property name="placeholderText">
       <string>请输入待加密文本...</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Encryption&lt;/p&gt;&lt;p&gt;Result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Encryption&lt;/p&gt;&lt;p&gt;Algorithm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="4">
     <widget class="QTextEdit" name="Enresult">
      <property name="placeholderText">
       <string>显示密文中...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QToolButton" name="EnFileButton">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Plaintext</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QComboBox" name="EnAlgorithm">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Caesar cipher</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Affine cipher</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multilateral cipher</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Keyword cipher</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vigenere cipher</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autokey ciphertext</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autokey plaintext</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Playfair cipher</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Permutation cipher</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Column permutation cipher</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rc4 cipher</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Key</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="EnButton">
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Encryption</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="4">
     <widget class="QPlainTextEdit" name="EnKeytext">
      <property name="placeholderText">
       <string>请输入密钥</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="SendSocket">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="EnFileline">
      <property name="placeholderText">
       <string>请选择文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
